lib/modules/manager/github-actions/extract.ts

```diff
@@ -68,6 +69,48 @@ export function extractPackageFile(content: string): PackageFile | null {
       deps.push(dep);
     }
   }
+  return deps;
+}
+
+function extractContainer(container: string | Container): PackageDependency {
+  let dep: PackageDependency;
+  if (typeof container === 'string') {
+    dep = getDep(container);
+  } else {
+    dep = getDep(container?.image);
+  }
+  return dep;
+}
+
+function extractWithYAMLParser(content: string): PackageDependency[] {
+  logger.trace('github-actions.extractWithYAMLParser()');
+  const deps: PackageDependency[] = [];
+
+  const pkg = load(content, { json: true }) as Workflow;
+
+  for (const job of Object.values(pkg.jobs ?? [])) {
```

Comment:
```suggestion
  for (const job of Object.values(pkg.jobs ?? {})) {
```

ups, should be an object instead of an array.